image-pipeline/expression-scatter-plots.js

```diff
@@ -21,7 +21,8 @@ import { Storage } from '@google-cloud/storage'
 let numLogEntries = 0
 
 /** Print message with browser-tag preamble to local console and log file */
-function print(message, preamble='') {
+function print(message, context={}) {
+  let preamble = ('preamble' in context === false) ? '' : context.preamble
```

Comment:
I'm curious the benefit to checking if 'preamble' is in the context object in this way. As far as I can tell this is essentially only guarding for a blank object but `context.preamble` could still be fals-y here, should we guard against those cases as well?

Reply:
I'm unaware of a case where `context.preamble` could be falsy, but I'm open to specific suggestions.